tools/gather-sysinfo/gather-sysinfo_test.go

```diff
@@ -0,0 +1,91 @@
+package main
```

Comment:
we may want to strip the logic from the main source and move them into an **internal** package to make unit test easier

Reply:
`tools/gather-sysinfo/$WHATEVER` can be a nice start

Reply:
ok, thanks